pkg/provenance.go

```diff
@@ -199,6 +199,9 @@ func GenerateProvenance(name, digest, ghContext, command, envs string) ([]byte,
 						"sha1": gh.SHA,
 					},
 				},
+				{
+					URI: fmt.Sprintf("https://github.com/actions/virtual-environments/releases/tag/%s/%s", os.Getenv("ImageOS"), os.Getenv("ImageVersion")),
+				},
```

Comment:
can we pull the sha256 for the release?
We can do that in a follow-up PR

Reply:
I'll file an issue to handle this as a follow-up